ci-operator/step-registry/ipi/install/heterogeneous/ipi-install-heterogeneous-commands.sh

```diff
@@ -239,6 +239,7 @@ case $CLUSTER_TYPE in
   infra_id=$(yq-v4 '.infraID' < "${SHARED_DIR}"/metadata.json)
   rg_name="${infra_id}-rg"
   sa_name=$(az storage account list -g "${rg_name}" | yq-v4 '.[] | select(.name == "cluster*").name')
+  [[ -z "${sa_name}" ]] && sa_name=$(az storage account list -g "${rg_name}" | yq-v4 ".[] | select(.name == \"${infra_id//-}*\").name")
```

Comment:
hi, @jinyunma, can you add some comments here to explain why we need this now for future readers?

Reply:
sure, updated.